IM_calculation/Advanced_IM/Models/Taghavi_Miranda_2005/run.py

```diff
@@ -0,0 +1,85 @@
+"""
+Python script to run a 3D waveform through Taghavi_Miranda_2005 and store the outputs to a txt file
+"""
+
+import argparse
+import glob
+import os
+import subprocess
+
+import numpy as np
+import pandas as pd
+
+from IM_calculation.IM.Taghavi_Miranda_2005 import Taghavi_Miranda_2005
+from qcore.timeseries import read_ascii
+
+model_dir = os.path.dirname(__file__)
+
+COMPONENT = ["000", "090"]
+ALPHA = [0,   1.5,   5,   15,   30]
+C = [0.02, 0.05, 0.1]
+T = [0.1, 0.4, 0.7, 1.5, 3.0]
+STORIES = 10
+
+
+def main():
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "comp_000", help="filepath to a station's 000 waveform ascii file"
+    )
+    parser.add_argument(
+        "comp_090", help="filepath to a station's 090 waveform ascii file"
+    )
+    parser.add_argument(
+        "comp_ver", help="filepath to a station's ver waveform ascii file"
+    )
+    parser.add_argument(
+        "output_dir",
+        help="Where the IM_csv file is written to. Also contains the temporary recorders output",
+    )
+    parser.add_argument(
```

Comment:
Remove and use parse_known_args, logging or printing any unused?
Or refactor advanced IMs to not pass it